tools/query_verification/src/main/java/com/google/bigquery/Main.java

```diff
@@ -0,0 +1,175 @@
+package com.google.bigquery;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Comparator;
+
+import org.apache.commons.cli.*;
+
+public class Main {
+
+    /**
+     * usage: query_verification -q <PATH> <PATH> [-d <PATH>] [-s <PATH> <PATH>]
+     *        [-h]
+     *  -q,--query <PATH> <PATH>    First argument is the path to the migrated
+     *                              query file. Second argument is the path to
+     *                              the original query file and only required
+     *                              when data is provided.
+     *  -d,--data <PATH>            Path for table data in CSV format.
+     *  -s,--schema <PATH> <PATH>   First argument is the path to the migrated
+     *                              schema path. Second argument is the path to
+     *                              the original schema query and is optional.
+     *                              Referenced files should be in a JSON format.
+     *  -h,--help                   Print this help screen.
+     *
+     * @param args Command line arguments
+     */
+    public static void main(String[] args) {
+        Main main = new Main();
+        main.run(args);
+
+        System.exit(0);
+    }
+
+    public void run(String[] args) {
+        CommandLine command = buildCommand(args);
+        if (command == null) {
+            return;
+        }
+
+        QueryVerification queryVerification = new QueryVerification();
```

Comment:
I think it might look cleaner if you restructure things so that original/migratedQuery and schema get assigned to some variables at this scope, then create the QueryVerification object using the complete constructor. Then you can remove the setters as well.